### PR TITLE
session-desktop 1.11.4 -> 1.11.5

### PR DESCRIPTION
**Pull request by ghost**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by TheDuckWithAMonocle** [comment]
Fixes #278735

File: pkgs/applications/networking/instant-messengers/session-desktop/default.nix [hunk 1]
```diff
@@ -8,7 +8,7 @@
 }:
 
 let
-  version = "1.11.4";
+  version = "1.11.5";
   pname = "session-desktop";
```
Comment (eclairevoyant) [inline pkgs/applications/networking/instant-messengers/session-desktop/default.nix:12]: please correct the hash for this

File: pkgs/applications/video/lbry/default.nix [hunk 2]
```diff
@@ -12,7 +12,7 @@ in appimageTools.wrapAppImage rec {
     src = fetchurl {
       url = "https://github.com/lbryio/lbry-desktop/releases/download/v${version}/LBRY_${version}.AppImage";
       # Gotten from latest-linux.yml
-      hash = "sha512-WZB2pMzSuWGPj6uad+rIECOhuWEOxi0hVUQifOrhUrKj4SnBDws+oy7V2+NpDGkzbG+Kf3IO8rcWBD4wfFoo2Q==";
```
Comment (eclairevoyant) [inline pkgs/applications/video/lbry/default.nix:15]: doesn't seem related, so updating this should be a separate PR

**Review by eclairevoyant — CHANGES_REQUESTED** [review]
Thanks for the contribution! Please update the PR title and some feedback below

File: pkgs/applications/networking/instant-messengers/session-desktop/default.nix (hunk 1 above)
Comment (TheDuckWithAMonocle) [reply pkgs/applications/networking/instant-messengers/session-desktop/default.nix:12]: Done.

**Comment by Alexnortung** [comment]
I think the commits should follow the commit conventions https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#commit-conventions
So you should probably make one commit for lbry and one for session or maybe even make two PRs

**Comment by eclairevoyant** [comment]
> please correct the hash for this

This was meant for session-desktop - the version was changed but the hash was not. Please use one of the methods mentioned [here](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#obtaining-source-hash).

**Comment by TheDuckWithAMonocle** [comment]
Sorry about that, i made more than a few mistakes. Let me finalise this, and you guys won't have to deal with this for any longer
